xAPI-About.md

```diff
@@ -525,6 +570,7 @@ Clarifications and additional examples including:
 ###### 1.0.2 to 1.0.3 (TBC, 2016)
 - Complete reorganization of the document into three parts
 - Many, many clarifications
+- Definitions slightly modified and greatly clarified
```

Comment:
Seems confusing to say they were only slightly modified but greatly clarified. Why not keep it simple and say "Revised some definitions."